packages/console/src/hooks/use-documentation-url.ts

```diff
@@ -6,7 +6,8 @@ import { useTranslation } from 'react-i18next';
  */
 enum DocumentationLanguage {
   English = 'en',
-  Chinese = 'zh-CN',
+  /** @deprecated zh-CN doc has been temporarily removed since it's too out of date, will bring it back in the future. */
+  // Chinese = 'zh-CN',
```

Comment:
can we just remove it for now? since it may take some time to bring back

Reply:
Updated.